dune-project

```diff
@@ -0,0 +1,2 @@
+(lang dune 0.1)
```

Comment:
Just to let you know, I just changed the version to `1.0` (https://github.com/ocaml/dune/pull/873). I was planning to do this change at some point before the 1.0.0 release of Dune, but someone told me about this PR and that it was relying on Dune master, so I thought it was better to do this change sooner rather than latter.

Reply:
Hi Jérémy, thanks for the heads up and for having a look, we will update the PR.

We still want to make a few tweaks to the PR but my intention is to submit a bit of feedback to Dune developers once we consider this PR ready.